src/libraries/System.Security.Cryptography.Algorithms/tests/ChaCha20Poly1305Tests.cs

```diff
@@ -456,5 +462,22 @@ public static void CheckIsSupported()
 
             Assert.Equal(expectedIsSupported, ChaCha20Poly1305.IsSupported);
         }
+
+        private static int GetAndroidSdkVersion()
```

Comment:
Does it make sense to move this to a shared helper library? If you don't think anybody else has an immediate need for it, disregard this comment.

Reply:
I'm not sure. I think the majority of tests / implementation should do feature detection (like how `IsSupported` is actually implemented). However we want to test the `IsSupported` implementation against some other mechanism for correctness, and in Android that would be the API version.

I hesitated to put this somewhere easily accessible off of `PlatformDetection` or similar, otherwise it might be mistaken as a good way to guard tests / functionality when we only need it here for a very specific purpose.

Reply:
Oh, you already fixed it. Thank you!